include/grpc++/client_context.h

```diff
@@ -64,8 +63,6 @@
 #include <grpc/support/log.h>
 #include <grpc/support/time.h>
 
-struct census_context;
```

Comment:
Same question: we want to have the context available from client/server, so people can get a reference to it and later use it for stats recording.